src/sentry/api/urls.py

```diff
@@ -2066,7 +2066,7 @@
         name="sentry-api-0-project-rule-details",
     ),
     url(
-        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/rules/(?P<rule_id>[^\/]+)/snooze/$",
+        r"^(?P<organization_slug>[^\/]+)/(?P<project_slug>[^\/]+)/rules/(?P<rule_id>[^\/]+)/(?P<rule_type>[^\/]+)/snooze/$",
```

Comment:
I think that this url structure might be a bit confusing. `rules/` so far specifically means just issue alert rules.

What could potentially work here is to keep this url  as is, treat most of the logic in `RuleSnoozeEndpoint` as `BaseRuleSnoozeEndpoint`, then we'd have
```
class RuleSnoozeEndpoint(BaseRuleSnoozeEndpoint):
    type = RuleType.ISSUE_ALERT
```
and for metric alerts (if/when we do that)

```
class MetricRuleSnoozeEndpoint(BaseRuleSnoozeEndpoint):
    type = RuleType.METRIC_ALERT
```

That way the type is just inferred from the url, and we don't change the structure for metric alerts.

Reply:
You mean the URLs would be like `/{org_slug}/{project_slug}/issue-alert-rule/{rule_id}/snooze` (and another for metric alerts)?

Reply:
Ok pushed my interpretation 😅 lmk if I misunderstood